.github/workflows/actions.yml

```diff
@@ -590,7 +590,7 @@ jobs:
 
           # install API quark
           mkdir $QUARKS_PATH && cd $QUARKS_PATH
-          git clone --depth=1 https://github.com/supercollider-quarks/API
+          git clone --depth=1 https://github.com/crucialfelix/API
```

Comment:
is there a repo (other than someone specifically, who could just destroy it) that we can use as a test? I don't think the crucial lib is going away, but it stood out a little

Reply:
Using the API quark is part of the whole qpm design (for using json files). I'm not a fan of using any quark for testing, but if we use one I think it's better if it's the currently maintained version.

IMO the long term solution is to re-consider using qpm and how we run tests to possibly not have to use a quark altogether.